frontend/src/pages/Graph/SummaryPanel.tsx

```diff
@@ -259,7 +259,7 @@ class SummaryPanelComponent extends React.Component<MainSummaryPanelPropType, Su
 }
 
 const mapStateToProps = (state: KialiAppState) => ({
-  jaegerState: state.jaegerState,
+  jaegerState: state.tracingState,
```

Comment:
If the state is generic (tracingState), the variable should be renamed to tracingState.